Bookish.Web/Models/NewBookViewModel.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Reflection;
+using Bookish.DataAccess;
+
+namespace Bookish.Web.Models
+{
+    public class NewBookViewModel
+    {
+        public string Title;
+        public string Author;
+        public string Isbn;
+        public int TotalCopies;
+
+        public NewBookViewModel(string title, string author, string isbn, int totalCopies)
+        {
+            Title = title;
+            Author = author;
+            Isbn = isbn;
+            TotalCopies = totalCopies;
```

Comment:
What if this is 0 or negative? 🙂

Reply:
The validation on the form should be able to stop that, unless users could somehow modify the request?

Reply:
It's best to assume that users can modify the request 🙂